### PR TITLE
Update footer to include new items [Fixes #1103]

### LINKED ISSUE #1103: Add two new footer items and three new hyperlinks
#### Current State of the Meshery.io site footer
Screenshot 2023-04-04 at 9 28 49 PM

#### Desired State of the Meshery.io site footer
Screenshot 2023-04-04 at 9 24 33 PM

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by anshgoyalevil**
**Description**
This PR adds new items to the footer using ```footer.yml```

This PR fixes #1103

**Notes for Reviewers**
Please review if it requires any changes.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by anshgoyalevil** [comment]
This PR can now be reviewed.

File: _includes/footer.html [hunk 1]
```diff
@@ -6,7 +6,13 @@
 			{% for column in site.data.footer %}
 			<ul class="footer-links">
 				<li>
-					<h2>{{ column.title }}</h2>
+					<h2><a {% if column.new_window %}target="_blank" {% endif %}
```
Comment (leecalcote) [inline _includes/footer.html:9]: Target should be self unless the link is to an external domain.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks, @anshgoyalevil 👍

**Review by leecalcote — APPROVED** [review]
Looks great. 💯

**Comment by leecalcote** [comment]
Yay, @anshgoyalevil! 🎈

**Pull request merged**